docs/courses/skill.md

```diff
@@ -128,13 +128,19 @@ Two-way-dictionary:
 - `Translation`: Translation of the phrase to the source language.
 - `Alternative translations`: A list of alternative translations to the source language, which will *not* be taught but will be accepted as correct translations of the `Phrase` field if supplied by the user. (optional)
 
-**`Mini-dictionary`** has a list of terms and meanings the user can view as a sort of "hint" if they are stuck. Entries are accessed by mousing over terms in an exercise, which brings up a tooltip with the corresponding term(s) in the other language. An entry is required for every term used in a skill.
+**`Mini-dictionary`** has a list of terms and meanings the user can view as a sort of "hint" if they are stuck. Entries are accessed by mousing over terms in an exercise, which brings up a tooltip with the corresponding term(s) in the other language. An entry is required for every term used in the composition of the phrases in a skill.
 
 Note that all `New words` are automatically added to the `Mini-dictionary`, in both translation directions, using the `Word` and `Translation` fields (`Synonyms` and `Also accepted` are ignored).
 
-`Mini-dictionary` entries do not only apply to this skill; the meanings will apply course-wide and be merged with the same term in other skills, if applicable. This means that entries can be omitted when they are already present in other skills.
```

Comment:
is this one true? or am I misunderstanding it? 

my understanding is that according to this, in the mini dictionary "building" and "Building" count as 2 different words? 

Actually I don't remember if this is true, but I'm surprised to hear it :thinking: